extension/js/common/core/attachment.ts

```diff
@@ -177,4 +177,62 @@ export class Attachment {
       return 'plainFile';
     }
   };
+
+  public isExecutableFile = () => {
+    return [
+      '.ade',
+      '.adp',
+      '.apk',
+      '.appx',
+      '.appxbundle',
+      '.bat',
+      '.cab',
+      '.chm',
+      '.cmd',
+      '.com',
+      '.cpl',
+      '.diagcab',
+      '.diagcfg',
+      '.diagpack',
+      '.dll',
+      '.dmg',
+      '.ex',
+      '.ex_',
+      '.exe',
+      '.hta',
+      '.img',
+      '.ins',
+      '.iso',
+      '.isp',
+      '.jar',
+      '.jnlp',
+      '.js',
+      '.jse',
+      '.lib',
+      '.lnk',
+      '.mde',
+      '.msc',
+      '.msi',
+      '.msix',
+      '.msixbundle',
+      '.msp',
+      '.mst',
+      '.nsh',
+      '.pif',
+      '.ps1',
+      '.scr',
+      '.sct',
+      '.shb',
+      '.sys',
+      '.vb',
+      '.vbe',
+      '.vbs',
+      '.vhd',
+      '.vxd',
+      '.wsc',
+      '.wsf',
+      '.wsh',
+      '.xll',
+    ].some(badFileExtension => this.name.endsWith(badFileExtension));
```

Comment:
that is `exeFileExtension => this.name.endsWith('.' + exeFileExtension)`

Reply:
done. thanks!